go.mod

```diff
@@ -117,3 +117,4 @@ require (
 )
 
 //replace github.com/ethereum/go-ethereum => github.com/certifaction/go-ethereum v1.10.3-wasm
+//replace github.com/herumi/bls-go-binary => github.com/0chain/bls-go-library-zboxmobile v1.26.1
```

Comment:
in gosdk/0chain, it uses `github.com/herumi/bls-go-binary`, not a fork. is it a problem if it is upgraded with latest upstream? I would suggest upstream first if we don't make any fixing on our fork.

Reply:
@cnlangzi For mobile we have to keep it from our forked repo, as there some changes. Legacy bls-go-binary is wrongly building binary for IOS and Mac